components/simulation-form/NewSimulationForm.tsx

```diff
@@ -32,6 +32,8 @@ import FormSection from './FormSection'
 import InterventionPeriodSection from './InterventionPeriodSection'
 import styles from './NewSimulationForm.module.css'
 import {SupportedParameters} from './SupportedParameters'
+import {ISODate} from '@covid-modeling/api/dist/src/model-input'
```

Comment:
Just `input` instead of referring to `dist`?

Reply:
Yes. Silly autocomplete added this without my seeing it.